### PR TITLE
Add new devices to Habitat

### PR DESCRIPTION
**Pull request by jimgao1**
This PR adds support for the following devices to Habitat:

- Desktop: RTX3090
- Datacenter/Workstation: A100, A40, A4000, RTX4000

### CODE REVIEW

**Review by johncalesp — COMMENTED** [review]
Looks great!, once it's merged, I will add the new card's information to the frontend

File: analyzer/habitat/analysis/mlp/dataset_process.py [hunk 1]
```diff
@@ -35,29 +35,24 @@ def get_dataset(path, features, device_features=None):
...
-
     print("Generating dataset")
     # generate vectorized dataset (one entry for each device with device params)
     device_params = get_all_devices(device_features)
 
     x, y = [], []
     for device in devices.keys():
-        df_merged_device = df_merged[features + [device, ]]
-        for row in tqdm(df_merged_device.iterrows(), leave=False, desc=device, total=len(df_merged_device.index)):
+        # df_merged_device = df_merged[features + [device, ]]
```
Comment (michaelshin) [inline analyzer/habitat/analysis/mlp/dataset_process.py:53]: you can remove this comment

File: experiments/run_experiment.py [hunk 2]
```diff
@@ -15,13 +15,20 @@
 # Experiment configuration
 
 DEVICES = [
-    habitat.Device.P4000,
-    habitat.Device.P100,
-    habitat.Device.V100,
-    habitat.Device.A100,
-    habitat.Device.T4,
-    habitat.Device.RTX2070,
+    # habitat.Device.P4000,
```
Comment (michaelshin) [inline experiments/run_experiment.py:18]: You can remove the commented devices

**Review by michaelshin — APPROVED** [review]
Looks good just added minor feedback

File: experiments/run_experiment.py (hunk 2 above)
Comment (jimgao1) [reply experiments/run_experiment.py:18]: Oh good catch. I actually didn't mean to remove it.

**Pull request merged**